### PR TITLE
Add deposit-request-completed webhook

### PR DESCRIPTION
**Pull request by jvc9109**
## Summary
Add option and specs for deposit completed webhook

## Checklist

- [x] Writing style
- [x] API design standards

### CODE REVIEW

**Comment by colinbird** [comment]
Should we also add deposit-request-initiated?  That's when the transaction id is available afaiu

File: openapi/components/schemas/GlobalEventType.yaml [hunk 1]
```diff
@@ -25,6 +25,8 @@ enum:
   - customer-returned
   - customer-updated
   - data-export-completed
+  - deposit-request-completed
+  - deposit-request-initiated
```
Comment (ashkarpetin) [inline openapi/components/schemas/GlobalEventType.yaml:29]:
```suggestion
  - deposit-request-initiated
  - deposit-request-completed
```

File: openapi/components/requestBodies/Webhooks/DepositRequestWebhook.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+            properties:
+              href:
+                description: Link URL.
+                type: string
+              rel:
+                description: Type of link.
+                type: string
+                enum:
+                  - deposit
+description: DepositRequest webhook request body resource.
```
Comment (ashkarpetin) [inline openapi/components/requestBodies/Webhooks/DepositRequestWebhook.yaml:43]:
```suggestion
description: Deposit request webhook request body resource.
```

File: openapi/components/requestBodies/Webhooks/DepositRequestWebhook.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+            type: object
+            properties:
+              href:
+                description: Link URL.
+                type: string
+              rel:
+                description: Type of link.
+                type: string
+                enum:
+                  - deposit
```
Comment (ashkarpetin) [inline openapi/components/requestBodies/Webhooks/DepositRequestWebhook.yaml:42]:
```suggestion
                  - deposit
                  - transaction
```

**Review by ashkarpetin — APPROVED** [review]

**Review by noon-dawg — APPROVED** [review]
Looks good on the docs side

**Comment by jvc9109** [comment]
not releasing the feature finally

**Pull request closed without merging**